OsmAnd-java/src/main/java/net/osmand/search/core/SearchCoreFactory.java

```diff
@@ -561,7 +639,8 @@ public boolean isCancelled() {
 								continue;
 							}
```

Comment:
Not possible to transfer
if (bb == null && !r.containsRouteData(x31, y31, x31, y31, 15)) {
continue;
} else if(bb != null && !r.containsRouteData(bb[0], bb[1], bb[2], bb[3], 15)) {
continue;
}